frontend/src/components/maps/leaflet/InfoSlideOut/HeaderActions.tsx

```diff
@@ -0,0 +1,122 @@
+import styled from 'styled-components';
+import * as React from 'react';
+import { Row } from 'react-bootstrap';
+import { IBuilding, IParcel, PropertyTypes } from 'actions/parcelsActions';
+import { useLeaflet } from 'react-leaflet';
+import { MAX_ZOOM } from 'constants/strings';
+import { Link, useLocation } from 'react-router-dom';
+import queryString from 'query-string';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import Claims from 'constants/claims';
+
+const LinkMenu = styled(Row)`
+  background-color: #f2f2f2;
+  height: 35px;
+  width: 341px;
+  margin-left: -10px;
+  font-size: 14px;
+  font-family: ProximaNova, sans-serif;
+  padding: 10px;
+  a {
+    padding: 0 10px;
+    color: #1a5a96;
+  }
+`;
+
+const VerticalBar = styled.div`
+  border-left: 6px solid rgba(96, 96, 96, 0.2);
+  height: 18px;
+  margin-left: 1%;
+  margin-right: 1%;
+  border-width: 2px;
+`;
+
+export type IHeaderActions = {
+  /** The selected property */
+  propertyInfo: IParcel | IBuilding | null;
+  /** the selected property type */
+  propertyTypeId: PropertyTypes | null;
+  /** additional action to be taken when a link in the menu is clicked */
+  onLinkClick?: () => void;
```

Comment:
I put this here since I wasn't sure if we wanted the pop out to close when the property detail view is open

Reply:
Yeah, not sure on that one either. @bbjornholt ?

Reply:
@agalbrai & @devinleighsmith - It's a good question. I don't think there's a case where we'd need to have both open at the same time. That would probably cramp the map size. My thought is that the pop-out panel should close it when the detail view is opened, but it could be opened manually even if the detail view is open. So it would close, but then the user could pop it open if they need to.